libusb/Makefile.am

```diff
@@ -27,9 +27,10 @@ OS_OPENBSD_SRC = os/openbsd_usb.c
 OS_SUNOS_SRC = os/sunos_usb.h os/sunos_usb.c
 OS_WINDOWS_SRC = libusb-1.0.def libusb-1.0.rc \
 		 os/windows_common.h os/windows_common.c \
```

Comment:
Any way you can add os/windows_hotplug.c to the Android.mk srcs (for libusb cc_library)? I believe there's an existing linker error when building this using mingw on linux.
>>ld.lld: error: undefined symbol: windows_start_event_monitor
>>> referenced by external/libusb/libusb/os/windows_common.c:570
>>>               out/soong/.intermediates/external/libusb/libusb/windows_x86_64_static/obj/exter
nal/libusb/libusb/os/windows_common.o:(windows_init)
<snip>
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
07:31:59 ninja failed with: exit status 1

Reply:
Meant to say: libusb/Android.bp(not Android.mk).

Reply:
Hello @shaju-mv , I am willing to help, but I am a little bit lost with your request. I am not familiar with building libusb-1.0 for android. I can not find libusb/Android.bp . Are you sure it is not a generated file?

Are you under Linux and using mingw to build Windows source to build libusb-1.0 for Android?

Reply:
> Meant to say: libusb/Android.bp(not Android.mk).

@shaju-mv 

This PR should have nothing to do with Android. Please look at the PR #1164 if you want to have better Android support.
* #1164 

Please state what you are trying to do. I will delete the comments if your comments have nothing to do with this PR.

Reply:
yes, I'm building on Linux targeting 64-bit Windows.
You can resolve this comment thread, since it's N/A (as you point out).